docs/source/algorithms/ConvertToReflectometryQ-v2.rst

```diff
@@ -0,0 +1,240 @@
+
+
+After Transformation
+####################
+
+You will usually want to rebin using :ref:`algm-BinMD` or
+:ref:`algm-SliceMD` after transformation because the output workspaces
+are not regularly binned.
+
+Binning Methods
+###############
+
+The *Method* property allows the binning method used when applying the
+coordinate transformations to be selected. The default method,
+*Centre*, takes center point of each input bin, and locates the
+corresponding output bin, adding the input bins value to it. Centre point rebinning is faster.
+
+*NormalisedPolygon* is a more sophisticated approach. It constructs
+a polygon using the boundaries of the input bin, then transforms that polygon
+into the output coordinates, and then searches for intersections with the
+output bins. The value added to each output bin is proportional to size of the
+overlap with the input bin. The normalised polygon approach gives better accuracy.
+
+Usage
+-----
+Normalised Polygon Transformation
+#################################
+
+**Example - Normalised Polygon transformation**
+
+.. testcode:: ExConvReflQSimple
+
+    workspace_name = "POLREF4699"
+    workspace_nexus_file = workspace_name + ".nxs"
+
+    Load(Filename=workspace_nexus_file,OutputWorkspace=workspace_name)
+    X = mtd[workspace_name]
+    X = ConvertUnits(InputWorkspace=X,Target="Wavelength",AlignBins="1")
+    # Reference intensity to normalise by
+    CropWorkspace(InputWorkspace=X,OutputWorkspace='Io',XMin=0.8,XMax=14.5,StartWorkspaceIndex=2,EndWorkspaceIndex=2)
+    # Crop out transmission and noisy data
+    CropWorkspace(InputWorkspace=X,OutputWorkspace='D',XMin=0.8,XMax=14.5,StartWorkspaceIndex=3)
+    Io=mtd['Io']
+    D=mtd['D']
+
+    # Perform the normalisation step
+    Divide(LHSWorkspace=D,RHSWorkspace=Io,OutputWorkspace='I',AllowDifferentNumberSpectra='1',ClearRHSWorkspace='1')
+    I=mtd['I'][0]
+
+    # Move the detector so that the detector channel matching the reflected beam is at 0,0
+    PIX = 1.1E-3 #m
+    SC = 75
+    avgDB = 29
+    zOffset = -PIX * ((SC - avgDB) * 0.5 + avgDB)
+    MoveInstrumentComponent(Workspace = I, ComponentName = "lineardetector", X = 0, Y = 0, Z = zOffset)
+
+    # Should now have signed theta vs Lambda
+    ConvertSpectrumAxis(InputWorkspace=I,OutputWorkspace='SignedTheta_vs_Wavelength',Target='signed_theta')
+
+    qxqy, vertexes_qxqy = ConvertToReflectometryQ(InputWorkspace='SignedTheta_vs_Wavelength', OutputDimensions='Q (lab frame)', Extents='-0.0005,0.0005,0,0.12', OutputAsMDWorkspace=False,Method='NormalisedPolygon')
+
+    kikf, vertexes_kikf = ConvertToReflectometryQ(InputWorkspace='SignedTheta_vs_Wavelength', OutputDimensions='K (incident, final)', Extents='0,0.05,0,0.05', OutputAsMDWorkspace=False,Method='NormalisedPolygon')
+
+    pipf, vertexes_pipf = ConvertToReflectometryQ(InputWorkspace='SignedTheta_vs_Wavelength', OutputDimensions='P (lab frame)', Extents='0,0.1,-0.02,0.15', OutputAsMDWorkspace=False,Method='NormalisedPolygon')
```

Comment:
This example code is identical to the example code in the v1 doc. should they de different to show the difference between the versions?